qemu/tests/numa_consistency.py

```diff
@@ -1,125 +1,102 @@
-
-    dd_size = 256
-    if dd_size * len(vcpu_threads) > int(params['mem']):
-        dd_size = int(int(params['mem']) / 2 / len(vcpu_threads))
-
-    mount_size = dd_size * len(vcpu_threads)
-
-    mount_cmd = "mount -o size=%dM -t tmpfs none /tmp" % mount_size
-
     qemu_pid = vm.get_pid()
-    drop = 0
-    for cpuid in range(len(vcpu_threads)):
-        error_context.context("Get vcpu %s used numa node." % cpuid,
-                              test.log.info)
-        memory_status, _ = utils_test.qemu.get_numa_status(host_numa_node,
-                                                           qemu_pid)
-        node_used_host = get_vcpu_used_node(host_numa_node,
-                                            vcpu_threads[cpuid])
-        node_used_host_index = node_list.index(node_used_host)
-        memory_used_before = memory_status[node_used_host_index]
-        error_context.context("Allocate memory in guest", test.log.info)
-        session.cmd(mount_cmd)
-        binded_dd_cmd = "taskset %s" % str(2 ** int(cpuid))
-        binded_dd_cmd += " dd if=/dev/urandom of=/tmp/%s" % cpuid
-        binded_dd_cmd += " bs=1M count=%s" % dd_size
-        session.cmd(binded_dd_cmd)
-        error_context.context("Check qemu process memory use status",
-                              test.log.info)
-        node_after = get_vcpu_used_node(host_numa_node, vcpu_threads[cpuid])
-        if node_after != node_used_host:
-            test.log.warn("Node used by vcpu thread changed. So drop the"
-                          " results in this round.")
-            drop += 1
-            continue
-        memory_status, _ = utils_test.qemu.get_numa_status(host_numa_node,
-                                                           qemu_pid)
-        memory_used_after = memory_status[node_used_host_index]
-        page_size = resource.getpagesize() / 1024
-        memory_allocated = (memory_used_after -
-                            memory_used_before) * page_size / 1024
-        if 1 - float(memory_allocated) / float(dd_size) > threshold:
-            numa_hardware_cmd = params.get("numa_hardware_cmd")
-            if numa_hardware_cmd:
-                numa_info = process.system_output(numa_hardware_cmd,
-                                                  ignore_status=True,
-                                                  shell=True)
-            msg = "Expect malloc %sM memory in node %s," % (dd_size,
-                                                            node_used_host)
-            msg += "but only malloc %sM \n" % memory_allocated
-            msg += "Please check more details of the numa node: %s" % numa_info
-            test.fail(msg)
-    session.close()
-
-    if drop == len(vcpu_threads):
-        test.error("All test rounds are dropped. Please test it again.")
+    try:
+        test_mem = float(params.get("mem")) * mem_ratio
+        guest_stress_args = "-a -p -l %sM" % int(test_mem)
```

Comment:
It's better to make these parameters configurable, like:
`guest_stress_args = params.get("guest_stress_args", "-a -p -l %sM")`

Reply:
@zhenyzha 
Good catch ! Updated now, thanks :grinning: